tools/build_rules/prelude_bazel

```diff
@@ -1 +1,2 @@
 load("@io_bazel_rules_scala//scala:scala.bzl", "scala_library", "scala_binary", "scala_test", "scala_repl")
+load("@io_bazel_rules_scala//scala:scala_import.bzl", "scala_import")
```

Comment:
I'm getting a bit -1 on prelude_bazel (until 0.9 and I haven't even checked there, it doesn't work great across remote repos). Can we not add this and use the `header` feature to add `scala_import` to the 3rdparty targets since it is only used there?

Reply:
Done